tests/utils/i18n.ts

```diff
@@ -0,0 +1,29 @@
+import { createIntl, createIntlCache } from "@formatjs/intl";
+import sharedEN from "@swan-io/shared-business/src/locales/en.json";
+import bankingEN from "../../clients/banking/src/locales/en.json";
+import onboardingEN from "../../clients/onboarding/src/locales/en.json";
```

Comment:
don't we have a risk of key collision here? we could use the `t` functions from each app directly in the tests to make sure there's no risk

Reply:
every bundle is prefixed by the client key on line 9 to avoid that

Reply:
oh right, missed that bit